### ISSUE TITLE
Uncaught `NullReferenceException` when persisting block 83024 on RC3 TestNet

### ISSUE THREAD

**Issue by spencercorwin** [neo-project/neo#2512]
**Describe the bug**
When trying to sync a local C# RC3 node with the TestNet there is an unhandled `NullReferenceException` on block 83024 (block hash `0x92fbb11ca58989a4194471d8d4ad1c8cd1152b15a8583cc25ef9e1929df2b754`). The exception comes from when the GasToken contract tries to burn the GAS of the sender of the only transaction in that block (account `0x08fd5ec2a9b92b1d2ee8722ac38db041cceb8122`). When it goes to burn the GAS for that account it looks up the account in storage and doesn't find it ([see here](https://github.com/neo-project/neo/blob/master/src/neo/SmartContract/Native/FungibleToken.cs#L102)). `storage` is `null` and the exception is thrown on the next line. This leads to a FAULT state, which then throws this exception [here](https://github.com/neo-project/neo/blob/master/src/neo/Ledger/Blockchain.cs#L387). Block 83024 is never persisted and it cannot sync past this block.

**To Reproduce**
1. Sync a Neo C# RC3 node with the TestNet
2. Wait until it gets to block 83024
3. It will sync headers after block 83024 but won't sync blocks because it can't persist block 83024

**Expected behavior**
Excepted no exception or exception to be handled.

**Screenshots**
None.

**Platform:**
 - OS: macOS 11.4 (Big Sur)
 - Version: `neo-cli` built from source at commit `d8baf615045038a59414f60c9f4b1ab64328509e` (rc3 commit)

**(Optional) Additional context**
I ran into this while syncing our NEO•ONE node with the RC3 TestNet. To debug it I synced a C# node with the RC3 TestNet but found the same problem. Perhaps I'm missing something or did something wrong, since obviously the TestNet is currently well past block 83024.

I can upload the folder with the synced blockchain data up to block 83024 if it helps someone to recreate this issue.

**Comment by superboyiii** [comment]
It's the height of when I make exec_fee_factor increased to 200, which cause consensus down because of no enough execution gas in verification. See here: https://github.com/neo-project/neo/issues/2492. I've reverted several blocks at that height.
Resync will solve this. If not, it means some of your peer nodes are also crashed at this height. Offline sync package will solve this, I've uploaded one today: https://packet.azureedge.net/neochain/neo3/chain.0.acc
